RecoLocalCalo/Configuration/python/hcalLocalRecoNZS_cff.py

```diff
@@ -1,40 +1,38 @@
 import FWCore.ParameterSet.Config as cms
 
-import RecoLocalCalo.HcalRecProducers.HcalSimpleReconstructor_hbhe_cfi
-hbherecoMB = RecoLocalCalo.HcalRecProducers.HcalSimpleReconstructor_hbhe_cfi.hbheprereco.clone()
+import RecoLocalCalo.HcalRecProducers.HBHEPhase1Reconstructor_cfi
+hbherecoMB = RecoLocalCalo.HcalRecProducers.HBHEPhase1Reconstructor_cfi.hbheprereco.clone(
+    dropZSmarkedPassed = cms.bool(False),
```

Comment:
was .clone not enough , else how to explain this problem 
https://hypernews.cern.ch/HyperNews/CMS/get/recoDevelopment/1522.html